### PR TITLE
Add no_proxy settings to use minikube and kubectl

### PR DESCRIPTION
**Pull request by ybr**
Add Virtual Machine IP address to no_proxy settings to direct access the VM thus by-passing the proxy.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4108)

### CODE REVIEW

File: docs/getting-started-guides/minikube.md [hunk 1]
```diff
@@ -279,6 +279,12 @@ $ minikube start --docker-env HTTP_PROXY=http://$YOURPROXY:PORT \
                  --docker-env HTTPS_PROXY=https://$YOURPROXY:PORT
 ```
 
+Your Virtual Machine address might be 192.168.99.100, chances are your proxy settings will prevent kubectl to directly reach it.
```
Comment (chenopis) [inline docs/getting-started-guides/minikube.md]: suggested edit: "If your Virtual Machine address is 192.168.99.100, then chances are your proxy settings will prevent kubectl from directly reaching it."

File: docs/getting-started-guides/minikube.md [hunk 2]
```diff
@@ -279,6 +279,12 @@ $ minikube start --docker-env HTTP_PROXY=http://$YOURPROXY:PORT \
...
+To by-pass proxy configuration for this IP address you should modify your no_proxy settings, you can do so with:
```
Comment (chenopis) [inline docs/getting-started-guides/minikube.md]: suggested edit: "To by-pass proxy configuration for this IP address, you should modify your no_proxy settings. You can do so with:"

File: docs/getting-started-guides/minikube.md (hunk 1 above)
Comment (ybr) [inline docs/getting-started-guides/minikube.md]: My apologies for this late response, I was waiting for an email from github to let me know about this pull request.

File: docs/getting-started-guides/minikube.md (hunk 2 above)
Comment (ybr) [inline docs/getting-started-guides/minikube.md]: Yes of course :)

**Comment by chenopis** [comment]
@r2d4 Are you available for a tech review?

**Comment by chenopis** [comment]
@r2d4 I don't see a good spot on that readme page where this could go. Do you have any suggestions?

**Comment by r2d4** [comment]
Ah sorry @chenopis, we split it out into a proxy specific section here:
https://github.com/kubernetes/minikube/blob/master/docs/http_proxy.md

**Pull request merged**